lib/http_server_fixture/handler/simple_post.ex

```diff
@@ -1,9 +1,14 @@
 defmodule HttpServerFixture.Handler.SimplePost do
+  alias HttpServer.Response.Headers
+
   @behaviour HttpServer.Handler
 
   @impl HttpServer.Handler
   def handle(request, builder) do
+    headers = %Headers{content_type: "text/plain"}
+
     builder.new_response()
+    |>builder.add_headers(headers)
```

Comment:
```suggestion
    |> builder.add_headers(headers)
```
![image](https://user-images.githubusercontent.com/26009042/147152513-6f02eee3-975b-4c73-879e-120e744da251.png)

Reply:
I am actually surprised the linter did not find this.

Reply:
It's possible that this rule is disabled. I'll take a look.

Reply:
No, the rule `Credo.Check.Consistency.SpaceAroundOperators, []` is active, but it looks like it checks space around operators like +, -, * and /. It's possible doesn't check if after the pipe operator.

Reply:
Sounds like an opportunity for an open source PR for Credo (assuming its an open source project).